src/Context/Video/Module/Notification/Application/Send/SendStaffNotificationOnNotificationCreated.php

```diff
@@ -23,7 +23,7 @@ public static function subscribedTo(): array
     public function __invoke(NotificationCreatedDomainEvent $event)
     {
         $text = new NotificationText($event->text());
-        $type = new NotificationType($event->type());
```

Comment:
We should isolate the pull requests in order to make it easier to review & faster to integrate. Please, remove any change not related to the Docker compose feature from this PR.

If you want, you can make other PRs with these changes in order to be reviewed. Thanks a lot!!!

Reply:
Yes, I know! But at the moment I don't know why this code is here, because is from a PR from @rgomezcasas, in my commit are only changes to the docker-compose